### ISSUE TITLE
[BUG] #161 undefined variable

### ISSUE THREAD

**Issue by olokos** [surftimer/Surftimer-olokos#166]
**Describe the bug**
Plugin should compile perfectly

**Expected behavior**
13 errors during compilation
**Timer Version**
ed03199
**Additional context**
```
surftimer/sql.sp(1416) : error 017: undefined symbol "IsPlayerVip"
surftimer/sql.sp(3359) : error 017: undefined symbol "IsPlayerVip"
surftimer/sql.sp(9400) : error 017: undefined symbol "IsPlayerVip"
surftimer/commands.sp(346) : error 017: undefined symbol "IsPlayerVip"
surftimer/commands.sp(492) : error 017: undefined symbol "IsPlayerVip"
surftimer/commands.sp(535) : error 017: undefined symbol "IsPlayerVip"
surftimer/commands.sp(3171) : error 017: undefined symbol "IsPlayerVip"
surftimer/commands.sp(3178) : error 017: undefined symbol "IsPlayerVip"
surftimer/commands.sp(3236) : error 017: undefined symbol "IsPlayerVip"
surftimer/commands.sp(3254) : error 017: undefined symbol "IsPlayerVip"
surftimer/commands.sp(3268) : error 017: undefined symbol "IsPlayerVip"
surftimer/commands.sp(3361) : error 017: undefined symbol "IsPlayerVip"
surftimer/commands.sp(3970) : error 017: undefined symbol "IsPlayerVip"

13 Errors.
##[error]Process completed with exit code 1.
```

@Sarrus1 
Can you handle this? :)

**Comment by Bara** [comment]
https://github.com/surftimer/Surftimer-olokos/compare/f875920092ca...ed0319989d49#diff-09998b4bf3cd6cd14e2f8e55d443d256L4656

**Comment by Sarrus1** [comment]
I'll take a look tonight! Is this because of my commit? I didn't even touch IsPlayerVip...

**Comment by Bara** [comment]
You removed it in this commit: https://github.com/surftimer/Surftimer-olokos/pull/161/commits/6bd2171566c02b3091af3a90e31bc41e7c4eece6

**Comment by Sarrus1** [comment]
Oh that was a mistake, sorry! I'll fix it right away then.

**Comment by Sarrus1** [comment]
> Oh that was a mistake, sorry! I'll fix it right away then.

Done

**Comment by olokos** [comment]
Fixed in #167
